CHANGELOG.md

```diff
@@ -7,9 +7,12 @@
     a username.
     In that case, this header used to only contain the local part of the user's email address (e.g. `john.doe` for `john.doe@example.com`) but now contains
     the user's full email address instead.
+- [#137](https://github.com/pusher/oauth2_proxy/pull/137) When using gitlab provider, the `validate-url` has to point the the `api/v4` endpoint instead of the `api/v4/user` endpoint*
```

Comment:
Does this really count as a breaking change since the change is internal and not a user config that has to change?

Reply:
I guess so. It is a setting, that has to be set for an on premise gitlab.

Reply:
There's a bad merge here, removing someone else's change note